index.html

```diff
@@ -139,10 +139,10 @@ <h3>Simple Budgeting</h3>
   <section class="partner">
     <h3>Partner</h3> <hr>
     <ul>
-      <li>Bank-Central</li>
-      <li>Guinea-Games</li>
-      <li>freeCodeCamp</li>
+      <li>Google</li>
+      <li>Central-Banking</li>
       <li>Microverse</li>
+      <li>freeCodeCamp</li>
     </ul>
   </section>
```

Comment:
- [OPTIONAL ] In the home page for the mobile version I think it would be great if you remove the partner section since in the given design there is no partner section so kindly remove them from the Home page in Mobile version 🙏🏻🔥🤩

- Your 
![image](https://github.com/aradradev/easyBankTC/assets/58553711/ca6178d1-059f-4693-9d2d-c6294c2b2704)


- Given
![image](https://github.com/microhass/capstone-1/assets/58553711/f45b23df-e9ec-4f57-8244-1ec0fa044cca)